client/client-multi/client-cpp/tools/env.bat

```diff
@@ -40,24 +40,23 @@ if not %BUILD_PLATFORM% == x86 (
   set BUILD_PLATFORM=x64
 )
 
+@REM Root path which will be passed to CMake during SDK compilation
+set ROOT_PATH=C:\local\
```

Comment:
Consider not using environment variables at all. Probably, all `*bat` files should be dropped. TBD.